central/alert/datastore/datastore_impl.go

```diff
@@ -260,6 +263,16 @@ func (ds *datastoreImpl) updateAlertNoLock(ctx context.Context, alerts ...*stora
 	if len(alerts) == 0 {
 		return nil
 	}
+
+	for _, alert := range alerts {
+		alert.EntityType = alertutils.GetEntityType(alert)
+		match, err := ds.platformMatcher.MatchAlert(alert)
+		if err != nil {
+			return err
+		}
+		alert.PlatformComponent = match
```

Comment:
An updated test with a mixture of platform and non-platform alerts would probably be a good thing.